src/GovUk.Education.ExploreEducationStatistics.Content.Api.Tests/Services/PublicationServiceTests.cs

```diff
@@ -0,0 +1,1584 @@
+#nullable enable
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using GovUk.Education.ExploreEducationStatistics.Common.Model;
+using GovUk.Education.ExploreEducationStatistics.Common.Tests.Utils;
+using GovUk.Education.ExploreEducationStatistics.Content.Api.Services;
+using GovUk.Education.ExploreEducationStatistics.Content.Api.Services.Interfaces;
+using GovUk.Education.ExploreEducationStatistics.Content.Model;
+using GovUk.Education.ExploreEducationStatistics.Content.Model.Database;
+using Moq;
+using Xunit;
+using static GovUk.Education.ExploreEducationStatistics.Common.Services.CollectionUtils;
+using static GovUk.Education.ExploreEducationStatistics.Content.Model.Database.ContentDbUtils;
+
+namespace GovUk.Education.ExploreEducationStatistics.Content.Api.Tests.Services
+{
+    public class PublicationServiceTests
+    {
+        [Fact]
+        public async Task GetPublicationTree_Empty()
```

Comment:
Was trying to find the equivalent tests that these were based on, but can't find them in this PR.  I guess then we didn't have a great deal of test coverage around this before?

If so, glad it's in now!

Reply:
Nope, only really basic coverage I'm afraid. Think it's a lot better now!